test/plugins/test/autolink.js

```diff
@@ -130,8 +129,14 @@ describe('AutoLink', function() {
 
 		testLink.call(this, {
 			expected:
-				'<p>invalid link <a href="http://ww.liferay.com">ww.liferay.com</a></p><p>text</p>',
-			html: '<p>invalid link ww.liferay.com</p>{ }<p>text</p>',
+				'<p>link <a href="http://ww.liferay.com">ww.liferay.com</a></p><p>text</p>',
+			html: '<p>link ww.liferay.com</p>{ }<p>text</p>',
+			keyCode: KEY_ENTER,
+		});
+
+		testLink.call(this, {
+			expected: '<p>invalid link eeeewwww</p><p>text</p>',
+			html: '<p>invalid link eeeewwww</p>{ }<p>text</p>',
```

Comment:
And this one would go in the same `it` along with the other actually-invalid one.